### PR TITLE
DCOS_OSS-4542: Add Delayed Status to service debug page

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Add a delayed status to the service debug screen.

Closes DCOS_OSS-4542

## Testing
1. Create a service with very high resource requirements.
Example JSON from the jira:
```
{
"id": "/delayed",
"backoffFactor": 10,
"backoffSeconds": 1,
"cmd": "exit 1",
"container": {
"type": "MESOS",
"volumes": []
},
"cpus": 0.1,
"disk": 0,
"instances": 10,
"maxLaunchDelaySeconds": 3600,
"mem": 10,
"gpus": 0,
"networks": [
{
"mode": "host"
}
],
"portDefinitions": [],
"requirePorts": false,
"upgradeStrategy": {
"maximumOverCapacity": 1,
"minimumHealthCapacity": 1
},
"killSelection": "YOUNGEST_FIRST",
"unreachableStrategy": {
"inactiveAfterSeconds": 0,
"expungeAfterSeconds": 0
},
"healthChecks": [],
"fetch": [],
"constraints": []
}
```
1.5 (Optional) Check the marathon UI `daily-cluster-url/service/marathon/ui/#/apps` to see that it is actually delayed.
2. Open it and go to the Debug tab.
3. Verify that the new message is shown.
4. Verify that the link is shown and it works (1.14 documentation is currently empty, but you can manually change the url to 1.13).
5. Verify the same for pods.
Example JSON:
```
{
  "id": "/delayed-pod",
  "version": "2019-05-28T11:20:07.349Z",
  "containers": [
    {
      "name": "container-1",
      "resources": {
        "cpus": 0.1,
        "mem": 10,
        "disk": 0,
        "gpus": 0
      },
      "exec": {
        "command": {
          "shell": "exit 1"
        }
      }
    }
  ],
  "networks": [
    {
      "mode": "host"
    }
  ],
  "scaling": {
    "instances": 10,
    "kind": "fixed"
  },
  "scheduling": {
    "backoff": {
      "backoff": 1,
      "backoffFactor": 1.15,
      "maxLaunchDelay": 300
    },
    "upgrade": {
      "minimumHealthCapacity": 1,
      "maximumOverCapacity": 1
    },
    "killSelection": "YOUNGEST_FIRST",
    "unreachableStrategy": {
      "inactiveAfterSeconds": 0,
      "expungeAfterSeconds": 0
    },
    "placement": {
      "constraints": []
    }
  },
  "executorResources": {
    "cpus": 0.1,
    "mem": 32,
    "disk": 10
  },
  "volumes": [],
  "fetch": []
}
```
6. Verify that the added tests make sense.

## Trade-offs
The link is missing from the design. @DanielMSchmidt said that he is going to talk to the designers about it, so the current link might change.

## Dependencies
https://github.com/dcos/dcos-ui/pull/3910

### Before
Screenshot from 2019-05-27 16-35-49

Screenshot from 2019-05-28 14-33-06

### After
Screenshot from 2019-05-28 15-13-59

Screenshot from 2019-05-28 14-32-27

### CODE REVIEW

File: plugins/services/src/js/containers/pod-debug/PodDebugContainer.js [hunk 1]
```diff
@@ -207,30 +208,54 @@ class PodDebugTabView extends React.Component {
       return null;
     }
 
+    let message, primaryAction;
+
+    if (this.props.pod.isDelayed()) {
```
Comment (DanielMSchmidt) [inline plugins/services/src/js/containers/pod-debug/PodDebugContainer.js]: The method name is now misleading, Delayed means not that it is waiting for resources.
I would suggest renaming the method to `getUnableToStartNotice`

Comment (GeorgiSTodorov) [reply plugins/services/src/js/containers/pod-debug/PodDebugContainer.js]: Do you mean the "isDelayed" method? It is not part of this PR, but I can create another PR to rename it.

**Comment by DanielMSchmidt** [comment]
No I mean the method name of this function in which you added the code I
commented on. Sorry for being unclear here

Georgi Todorov <notifications@github.com> schrieb am Di., 28. Mai 2019,
14:50:

**Comment by GeorgiSTodorov** [comment]
@DanielMSchmidt , ah, ok.

**Comment by GeorgiSTodorov** [comment]
@DanielMSchmidt , I change the names.

**Comment by GeorgiSTodorov** [comment]
Failing test is passing locally, retriggering.

**Comment by nLight** [comment]
Shall we run the re-run to make sure those not flaky?

**Comment by GeorgiSTodorov** [comment]
@nLight , you mean to test that one test and see if it is flaky? Daniel had a PR for that.

**Comment by nLight** [comment]
if that is an integration test we could `it.only` it on a sep PR. Just saw a lot of `flaky` word in the emails recently 🙈

**Comment by GeorgiSTodorov** [comment]
@nLight , we did: https://github.com/dcos/dcos-ui/pull/3849

**Comment by nLight** [comment]
oh, ok :) please ignore me :)

**Comment by GeorgiSTodorov** [comment]
@nLight , sorry for not being clearer in the first place.

**Pull request merged**